codegen/res/rust/common/common_types/options.rs

```diff
@@ -0,0 +1,58 @@
+/* This file will be copied into the folder for generated code. */
+
+use std::collections::HashMap;
+
+use derive_builder::Builder;
+
+#[allow(unused)]
+#[derive(Builder, Clone)]
+pub struct CommandExecutorOptions {
+    /// Optional Topic namespace to be prepended to the topic pattern
+    #[builder(default = "None")]
+    pub topic_namespace: Option<String>,
+    /// Topic token keys/values to be replaced in the topic pattern
+    #[builder(default)]
+    pub topic_token_map: HashMap<String, String>,
+}
+
+#[allow(unused)]
+#[derive(Builder, Clone)]
+pub struct CommandInvokerOptions {
+    /// Optional Topic namespace to be prepended to the topic pattern
+    #[builder(default = "None")]
+    pub topic_namespace: Option<String>,
+    /// Topic token keys/values to be replaced in the topic pattern
+    #[builder(default)]
+    pub topic_token_map: HashMap<String, String>,
+    /// Prefix for the response topic.
```

Comment:
```suggestion
    /// Prefix for the response topic.
    /// If all response topic options are `None`, the response topic will be generated
    /// based on the request topic in the form: `clients/<client_id>/<request_topic>`
```
just to match the doc strings in the invoker.rs struct